spec/views/vms/index.html.erb_spec.rb

```diff
@@ -65,16 +65,16 @@
 
       it 'shows no power buttons' do
         rendered = render
-        expect(rendered).not_to have_css('a.btn-manage.play')
-        expect(rendered).not_to have_css('a.btn-manage.stop')
+        expect(rendered).not_to have_css('a.fa-play')
+        expect(rendered).not_to have_css('a.fa-stop')
```

Comment:
Instead of checking for the presence of a css class, it might be more appropriate to check if a `href` is present, i.e. `a[href="#{restart_vm_path}"] `

`expect(page).to have_link(nil, href: 'actual link')` or 
`expect(rendered).to have_selector(:css, 'a[href="actual link"]')`

Reply:
This should be addressed once https://github.com/hpi-swt2/vm-portal/pull/464 is merged, as it will be easier to assign sane path names.